examples/governance/tests/BaselineGovernorTest.sol

```diff
@@ -0,0 +1,258 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import {Test} from "forge-std/Test.sol";
+import {GovernorTestBase} from "./GovernorTestBase.sol";
+import {console2} from "forge-std/console2.sol";
+import {BaselineGovernor} from "../contracts/BaselineGovernor.sol";
+import {VoteToken} from "../contracts/VoteToken.sol";
+import {IGovernor} from "openzeppelin/contracts/governance/IGovernor.sol";
+
+contract BaselineGovernorTest is Test, GovernorTestBase {
+    function setUp() public override {
+        super.setUp();
+    }
+
+    function testProposalCreation() public {
+        (
+            address[] memory targets,
+            uint256[] memory values,
+            bytes[] memory calldatas,
+            string memory description
+        ) = _createProposalParams();
+
+        uint256 proposalId = baselineGovernor.propose(
+            targets,
+            values,
+            calldatas,
+            description
+        );
+
+        assertGt(proposalId, 0, "Proposal should be created with non-zero ID");
+        assertEq(
+            uint256(baselineGovernor.state(proposalId)),
+            uint256(IGovernor.ProposalState.Pending),
+            "New proposal should be in Pending state"
+        );
+    }
+
+    function testProposalThreshold() public {
```

Comment:
Is the name of this test correct? It looks like this test is actually checking that different proposals have different IDs.